### DISCUSSION TITLE
Allow applying tape bindings (and maybe ribbon) from hand held tape roll

### DISCUSSION THREAD

**Discussion by Dwfreegethub** [littlesera/LSCG#201] [Ideas]
Review to see if you can do the same thing with the tape as you can with the Rope and the Ball gag. And not sure if there is a ribbon for the hand but if so the ribbon as well.  These should be able to use most of the same code as the rope tie.  

Please be sure to take a look at the Funnel gag and allow someone to dump the potions right in someone's mouth. 

There are some people who have a sleeping pill from one of the club games. Anyway to recognize that and use it to put someone to sleep if used on someone.  Maybe you could make it different from the other sleep items by making it longer to react, but last way longer..

**Comment by littlesera** [comment]
Tape is a good catch to include in the future. Not sure about ribbon, unless you maybe craft the tape and it switches on that.

The funnel I think is a bit of a BC bug, where if the funnel attached its MouthOpen function returns false. Ring gag has a similar issue, there's an override attempt in place but may need tuning.

IIRC there already was a mod that used the sleeping pills so I didn't initially plan anything for it, but that mod may be deprecated so I'll look into it.

**Comment by Dwfreegethub** [comment]
I don't want to open a second ticket but I have a niche case suggestion that you may or may not be able to do.   When my master kneels down it is my job to grab a cussion for him and bring it to him and either offer it to him or place it under his knees.. I am not sure how much this would be used outside of our play.. 

Being the cussion is one of those items that doesn't attach to hands I can't give it to him or place it on him using any of the LSCG. Is that something you would be willing to look into. Maybe once the current bugs are all squashed?

**Comment by littlesera** [comment]
Unfortunately, this will probably not be added, or at least not for a while. The mod can't really automate every RP need so I'm prioritizing larger generic use cases for now. Leaving this ticket open and renaming it for clarity.

**Comment by Dwfreegethub** [comment]
I totally get it. I just tossing ideas, I did not suspect that one would be very likely unless you though it would be super simple to add..

Thanks for replying.
